cadquery/freecad_impl/__init__.py

```diff
@@ -141,4 +141,4 @@ def _fc_path():
         import FreeCAD
 
 # logging
-import console_logging
+from . import console_logging
```

Comment:
good catch, I've really got to pay more attention when I'm importing internal modules.